src/main/java/com/fryrank/controller/UserMetadataController.java

```diff
@@ -9,11 +9,7 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.validation.BeanPropertyBindingResult;
 import org.springframework.validation.BindingResult;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestParam;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
```

Comment:
Let's not use star imports so it's clear which items are being imported specifically. You can also turn off the star import setting in IntelliJ

Reply:
I keep forgetting to turn it off 🤦‍♂️ , turned it off finally.

Reply:
Done in [98ac0c5](https://github.com/NickPriv/FryRankBackend/pull/83/commits/98ac0c5d7f9c27e6e75a242c815fd2e47c9cbdb6).